### PR TITLE
cleanup: remove useless codes when checking Skip status

### PR DESCRIPTION
**Pull request by kerthcet**
Signed-off-by: kerthcet <kerthcet@gmail.com>

#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/scheduler/framework/runtime/framework.go [hunk 1]
```diff
@@ -1038,7 +1038,7 @@ func (f *frameworkImpl) RunBindPlugins(ctx context.Context, state *framework.Cyc
 	}
 	for _, bp := range f.bindPlugins {
 		status = f.runBindPlugin(ctx, bp, state, pod, nodeName)
-		if status != nil && status.Code() == framework.Skip {
+		if status.Code() == framework.Skip {
```
Comment (denkensk) [inline pkg/scheduler/framework/runtime/framework.go:1041]: Add a member function for `status.IsSkip()` ?

Comment (kerthcet) [reply pkg/scheduler/framework/runtime/framework.go:1041]: This is the only place we used to verify the Skip code, so I prefer leave it is. And maybe add `IsSkip` method one day we have redundant codes.

Comment (denkensk) [reply pkg/scheduler/framework/runtime/framework.go:1041]: not for reducing the redundant codes. We have `IsSuccess` `IsUnschedulable` `IsWait`. Add `IsSkip` is to ensure consistent code formatting.

Comment (kerthcet) [reply pkg/scheduler/framework/runtime/framework.go:1041]: sg.

Comment (Huang-Wei) [reply pkg/scheduler/framework/runtime/framework.go:1041]: > Add a member function for status.IsSkip() ?

I recalled recently there is an issue discussing this, but cannot find the issue number..

**Comment by kerthcet** [comment]
PTAL @denkensk

**Comment by alculquicondor** [comment]
What code are you removing? It doesn't sound like the PR description is accurate.
In any case, I'll leave this review to @Huang-Wei, who is already involved.

**Comment by alculquicondor** [comment]
sounds good. The title is still very generic.

**Pull request merged**